game-library/game.py

```diff
@@ -66,17 +58,17 @@ def __init__(self, bot):
         game: Name of the game
         """
 
-        if user and not await check_permissions(ctx, {"manage_messages": True}):
```

Comment:
Why not put the entire block with the error message into the function?

Reply:
I can move the message there, but I'd still need the return here